packages/from-to-location-picker/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@opentripplanner/from-to-location-picker",
+  "version": "0.0.1",
+  "description": "Location Picker Component",
+  "author": "@binh-dam-ibigroup",
+  "homepage": "https://github.com/opentripplanner/otp-ui/#readme",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "module": "src/index.js",
+  "private": false,
+  "dependencies": {
+    "@opentripplanner/location-icon": "0.0.1"
```

Comment:
This should be updated to `^0.0.2`. Also, an entry for `prop-types` needs to be added here.

Reply:
Does this require merging with add-itineary-eas?

Reply:
It appears that that branch has been merged into master and there are now conflicts. So merging master into this branch should take care of all that.

Reply:
I merged with add-itinerary-eas which has everyone's most recent files and versions.